### DISCUSSION TITLE
Shop status?

### DISCUSSION THREAD

**Discussion by Levi--G** [john30/ebusd#1297] [General]
Hi, so i'm wondering about the adapter v5 boards (https://ebusd.eu/myadapter/), are they in stock right now? or are they still in preorder? If they are in stock i'd like to buy one but otherwise i could build a circuit myself as well. It wasn't really clear from the info on the site?

Thank you!

**Comment by tams9** [comment]
Hi, I would like to join! I would like to order a shield, if on stock. Is there an estimated schedule for manufacturing a new batch?

**Comment by JonesGW-MGD** [comment]
If you look on the the ebusd.eu website, you will see there are currently some adapters 'in stock' at Elecrow, although the drop-down says next shipping early August:

https://ebusd.eu/myadapter/

**Reply by yankee42** [reply]
For me the shop says
> Availability: Out of stock 

However the website says I cannot pre-order, because they are still in stock. Maybe the state just changed and needs to be updated on the website?

**Reply by Guzzi-Charlie** [reply]
Same for me. Would be interested but it's not possible to order (out of stock) and it's not possible to pre-order. What is the status?

**Reply by BenjaminK87** [reply]
Would also be interested.

**Reply by T-T-8844** [reply]
Hi. I'm also interested.

**Comment by SHAREN** [comment]
Hi, can I order an adapter to Russia?
